tests/test_connector.py

```diff
@@ -406,7 +409,7 @@ def test__handle_get_object_with_pagination_with_record(self):
                         "_return_as_object": 1,
                         "_max_results": 100}
         self.connector._get_object = mock.MagicMock(
-                                         return_value={"result": ["data"]})
```

Comment:
isnt this more redable

Reply:
Yes, but then Flake8 throws an error because the line is too long. I will add a disable flag.